app/build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
 apply plugin: 'com.android.application'
 apply plugin: 'me.tatarka.retrolambda'
 apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'android-apt'
```

Comment:
Why is this needed? Also, you could replace any 3rd party `apt` library with `annotationProcessor` that is in Android Gradle plugin 2.2+. See [this](https://bitbucket.org/hvisser/android-apt/wiki/Migration)